ci/prow/Makefile

```diff
@@ -33,8 +33,13 @@ config:
 
 update-config:
 	$(SET_CONTEXT)
+	$(eval TMP_YAML_CONFIG := $(shell mktemp))
+	kubectl get configmap config -o jsonpath="{.data}" 2>/dev/null >"${TMP_YAML_CONFIG}"
+	sed -i -e "s/((COMMIT_HASH_TOKEN))/$(shell git rev-parse HEAD)/" "config.yaml"
```

Comment:
Copy config.yaml to a temp file so you don't have to git checkout it (and also eliminate the dependency on git).

Reply:
good idea, done. Unfortunately can't completely eliminate dependency on git since we need the git commit